### ISSUE TITLE
Incorrect use of loss function

### ISSUE THREAD

**Issue by shurjobanerjee** [avdmitry/rl_3d#1]
I think the loss function has been used incorrectly. 
According to the documentation: https://www.tensorflow.org/api_docs/python/tf/losses/mean_squared_error
the function takes in the input: tf.losses.mean_squared_error(labels=?, predictions=?)

But in your code you use:
tf.losses.mean_squared_error(self.q, self.q_) which is in the opposite order.

**Comment by avdmitry** [comment]
Thank you for pointing on this, I fixed it. Result in both cases should be equal (as it squared), but anyway it is better to follow the documentation exactly.
